content/millennium/dstu2/medications/medication-order.md

```diff
@@ -79,8 +79,17 @@ _Implementation Notes_
  `patient`                 | Y         | [`reference`] | The identity of a patient to list orders for. Example: `12345`
  `status`                  | N         | [`token`]     | The status of the medication order, may be a list separated by commas. Example: `active,draft`
  `-timing-boundsPeriod`    | N         | [`date`]      | The date-time which should fall within the [period] the medication should be given to the patient. Must be prefixed by 'ge'. Example: `ge2014-05-19T20:54:02.000Z`
+ `_lastUpdated`            | N         | [`date`]      | The date-time within which the most recent clinically relevant update was made to the medication. Must be prefixed by 'ge' or 'le'. Example: `ge2014-05-19T20:54:02.000Z`
```

Comment:
I think 'within' is confusing as written for single instances of _lastUpdated.  What about something like:

"An explicit or implied date-time range within which the most recent clinically relevant update was made to the medication."